Templates/HowTo-template.md

```diff
@@ -0,0 +1,50 @@
+# How To [achieve result]
+
+
+Applicable Roles: [Identify developer roles/personas that find this type of information useful.]
+
+# Introduction
+The purpose of this procedure is to [do something] to [achieve result]. [Why is this procedure important?}
+
+Use this procedure [when] under [what circumstances]
+
+The expected result when you [use, launch, add, …] this procedure is [result].
+
+# Prerequisites
+Complete the following tasks prior to [using, launching, adding …] the [feature name]: <li> 
```

Comment:
remove \<li> at the end and just add:
\- ...
\- ...
The markdown parser will create the bullets for you.